src/main/java/io/pravega/connectors/hadoop/PravegaClientConfig.java

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.connectors.hadoop;
+
+import com.google.common.base.Preconditions;
+import io.pravega.client.ClientConfig;
+import io.pravega.client.stream.impl.Credentials;
+import lombok.Data;
```

Comment:
This import should be after those starting from `j`

Reply:
import order was influenced by the IDE but I fixed this one.